src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -860,36 +857,36 @@
+  "event.minecolonies.raidmessage_p1": "§4Mysterious sailors have been spotted close to %s of %s! They sailed in on a small sloop, you should consider destroying it.",
+  "event.minecolonies.raidmessage_p2": "§4Citizens report rumors of pirates close to %s of %s! They brought a ship, better destroy it.",
+  "event.minecolonies.raidmessage_p3": "§4Arr Comrade! There's some darn pirates from %s comin' for %s tonight! They sailed here and will continue attacking until we destroy their caravel.",
+  "event.minecolonies.raidmessage_p4": "§4A huge horde of pirates are sailin' %s of %s and they're not here to play! Their rigorous waves will hit us unless we destroy their galleon!",
 
-  "com.minecolonies.coremod.gui.workerhuts.toggleallon": "Toggle all (on)",
-  "com.minecolonies.coremod.gui.workerhuts.togglealloff": "Toggle all (off)",
+  "com.minecolonies.coremod.gui.workerhuts.toggleallon": "Toggle all on",
+  "com.minecolonies.coremod.gui.workerhuts.togglealloff": "Toggle all off",
   "com.minecolonies.coremod.gui.recipe.remove": "Remove",
-  "com.minecolonies.coremod.gui.workerhuts.crafting": "Teach recipe",
+  "com.minecolonies.coremod.gui.workerhuts.crafting": "Teach Recipe",
   "com.minecolonies.coremod.gui.recipe.done": "Successfully saved recipe!",
-  "com.minecolonies.coremod.gui.workerhuts.listrecipes": "List of recipes",
+  "com.minecolonies.coremod.gui.workerhuts.listrecipes": "List of Recipes",
   "com.minecolonies.coremod.resolvers.crafter.private": "Crafting",
-  "com.minecolonies.coremod.gui.workerhuts.togglereplantsaplingson": "Replant (on)",
-  "com.minecolonies.coremod.gui.workerhuts.togglereplantsaplingsoff": "Replant (off)",
+  "com.minecolonies.coremod.gui.workerhuts.togglereplantsaplingson": "Replant on",
+  "com.minecolonies.coremod.gui.workerhuts.togglereplantsaplingsoff": "Replant off",
   "com.minecolonies.coremod.resolvers.crafter.public": "Crafter",
 
   "com.minecolonies.coremod.compat.jei.crafting.teaching.unknown.recipe": "You have not learned this recipe yet!",
 
   "com.minecolonies.coremod.gui.townhall.allies": "Allies:",
   "com.minecolonies.coremod.gui.townhall.feuds": "Feuds:",
-  "com.minecolonies.coremod.gui.townhall.tp": "Do you really want to teleport to: %s [YES]",
+  "com.minecolonies.coremod.gui.townhall.tp": "Do you really want to teleport to %s? [YES]",
   "com.minecolonies.coremod.gui.workerhuts.switchstyle": "Switch Style",
-  "com.minecolonies.coremod.gui.townhall.toolow": "TH too low",
-  "com.minecolonies.coremod.gui.recipe.full": "Max reached for level",
-  "com.minecolonies.coremod.cook.full": "Can't bring food to the worker since their inventory is full.",
+  "com.minecolonies.coremod.gui.townhall.toolow": "Town Hall level too low!",
+  "com.minecolonies.coremod.gui.recipe.full": "Max number of recipes reached for hut level",
+  "com.minecolonies.coremod.cook.full": "I can't bring food to %s since their inventory is full!",
```

Comment:
This one I like. I'll help make sure the code is ready for it.

Reply:
Ok, hard to make the code ready for it when it doesn't appear to be used. It looks like at some point the cook was updated to 'eat on behalf' of a full citizen, destroying the food from his inventory and forcing saturation up on the citizen.

Reply:
Should I delete this line then?

Reply:
No, there is still a constant definition for it. I'll add it to my list to go clean this up.

Reply:
Added it back